### PR TITLE
Feature/pagination

### PR DESCRIPTION
**Pull request by annashamray**
fixes https://github.com/maykinmedia/objects-api/issues/148

### CODE REVIEW

File: src/objecttypes/api/views.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ class ObjectVersionViewSet(NestedViewSetMixin, viewsets.ModelViewSet):
     queryset = ObjectVersion.objects.order_by("object_type", "-version")
     serializer_class = ObjectVersionSerializer
     lookup_field = "version"
+    pagination_class = None
```
Comment (annashamray) [inline src/objecttypes/api/views.py:41]: I didn't add pagination to the nested resource, cause in Zaken API nested resources (zaakbesluiten, zaakeigenschappen) are not paginated. 
@joeribekker is it ok or should I paginate it?

Comment (joeribekker) [reply src/objecttypes/api/views.py:41]: do not paginate nested resources

**Review by joeribekker — COMMENTED** [review]
Approved but breaking, so let's wait for the end of sprint 7.

File: src/objecttypes/api/views.py (hunk 1 above)
Comment (annashamray) [reply src/objecttypes/api/views.py:41]: ok, thanks!

**Comment by annashamray** [comment]
implemented in https://github.com/maykinmedia/objecttypes-api/pull/59

**Pull request closed without merging**